tuna/flask.py

```diff
@@ -71,23 +70,27 @@ def get_tag_data(grafana_req, data):
     try:
       if query_tags != ['']:
         LOGGER.info(query_tags)
+        # pylint: disable=no-member ; @alex conv_config table has no column 'fusion_mode'
```

Comment:
ConvolutionConfig has no member 'fusion_mode'. But it might be some foreign key join thing going on...?

Reply:
there is no fusion_mode in ConvolutionConfig, this needs to be removed